_pages/en_US/seedminer.txt

```diff
@@ -22,11 +22,15 @@ While this test is not strictly necessary to perform the Seedminer exploit, the
 
 There are several different methods that build on the Seedminer exploit, so even if this test fails you can still use another method.
 
+1. With your system powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), then press (Power)
+  + Keep holding the buttons until your console boots into Safe Mode
+  + If your console does not boot into safemode or one of these buttons are broken you will need to use a [Legacy Method](legacy-methods)
```

Comment:
If one of the buttons is broken or the console does not boot into Safe Mode, you will need to use a legacy method at the bottom of this page (Do not link to Legacy Methods at this step -- at this part of the page, they still need to mine `movable.sed`)